skopt/dummy_opt.py

```diff
@@ -4,16 +4,19 @@
 import inspect
 import numbers
 import numpy as np
+from time import time
 
 from collections import Iterable
 from scipy.optimize import OptimizeResult
 from sklearn.utils import check_random_state
 
 from .space import Space
-
+from .utils import create_result
+from .utils import verbose_func
```

Comment:
We can have a naming improvement issue, where we resolve all such issues just before the release. (including the project name ;) )